IsogeomGenerator/generate_isogeom.py

```diff
@@ -0,0 +1,332 @@
+import argparse
+import os
+import vol as v
+
+
+def set_level_options(parser, moab):
+    """Sets options for specifying level values. If moab mode, do not
```

Comment:
Non sure what PEP8 saids about this, but I like to break the line here. I find it easier to read the comment

Reply:
So it turns out that all docstrings should start with a single line summary that is on the same line as the first set of quotes followed by a blank line and then a more in-depth description: https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings (this is what the PEP8 guide redirects to on docstrings)